### PR TITLE
[bn]: Translation fix

### PR DESCRIPTION
**Pull request by amdad121**
Some of translation bug fix.

### CODE REVIEW

File: locales/bn/php.json [hunk 1]
```diff
@@ -77,11 +77,11 @@
...
     "mimetypes": ":Attribute এর একটি ফাইল হতে হবে: :values।",
-    "min.array": ":Attribute অবশ্যই :min উপাদানের চেয়ে ছোট হতে হবে।",
-    "min.file": ":Attribute অবশ্যই :min কিলোবাইট এর চেয়ে ছোট হতে হবে।",
-    "min.numeric": ":Attribute অবশ্যই :min এর চেয়ে ছোট হতে হবে।",
-    "min.string": ":Attribute অবশ্যই :min অক্ষর এর চেয়ে ছোট হতে হবে।",
-    "min_digits": "এই :attribute এ সর্বোনিম্ন :min ডিজিট হতে হবে।",
+    "min.array": ":Attribute অবশ্যই :min উপাদানের চেয়ে ছোট হতে পারবে না।।",
+    "min.file": ":Attribute অবশ্যই :min কিলোবাইট এর চেয়ে ছোট হতে পারবে না।।",
+    "min.numeric": ":Attribute অবশ্যই :min এর চেয়ে ছোট হতে পারবে না।।",
+    "min.string": ":Attribute অবশ্যই :min অক্ষর এর চেয়ে ছোট হতে পারবে না।।",
```
Comment (arman-arif) [inline locales/bn/php.json:83]: Same problem here.
 "The :attribute must be at least :min characters."

File: locales/bn/php.json [hunk 2]
```diff
@@ -77,11 +77,11 @@
...
+    "min_digits": "এই :attribute এ সর্বোনিম্ন :min ডিজিট হতে পারবে না।।",
```
Comment (arman-arif) [inline locales/bn/php.json:84]: Same problem here.
"The :attribute field must have at least :min digits."

**Review by arman-arif — CHANGES_REQUESTED** [review]
Please update the translation, with the correct sentence.

**Comment by amdad121** [comment]
> Please update the translation, with the correct sentence.

Thank you. I have updated.

**Review by arman-arif — APPROVED** [review]

**Comment by arman-arif** [comment]
Thank you, It looks fine now.

**Comment by andrey-helldar** [comment]
@amdad121, @arman-arif, তোমাকে ধন্যবাদ

Released: https://github.com/Laravel-Lang/lang/releases/tag/15.9.0

**Pull request merged**